docs/static-header-footer/site-assets/x-bootstrap-4.css

```diff
@@ -259,7 +259,7 @@ a.btn, .nav a {
 }
 #wb-info .brand object, #wb-info .brand img {
 	height: 40px;
-	margin-bottom: 10px;
+	margin-bottom: 2em;
```

Comment:
I don't think the spacing should be based on text heights here, but rather a fixed pixels spacing instead.

Also, this is not the main file on which the change should be made. You can keep it here but make sure that you also make the change [over here in the footer _base.scss file](https://github.com/wet-boew/GCWeb/blob/a2a15774999219d70f47489c488d7962df250999/sites/footers/_base.scss#L26).